aws/lambda/log-classifier/ruleset.toml

```diff
@@ -183,3 +191,7 @@ pattern = 'An unexpected error has occurred. Conda has prepared the above report
 [[rule]]
 name = 'GHA error'
 pattern = '^##\[error\](.*)'
+
+[[rule]]
+name = 'Pytest Failure from summary'
+pattern = '^[-:\.\d+Z\s]* (FAILED .*)'
```

Comment:
i dont think this is needed? suo already added pytest failure and afaik there are no cases where it hasnt worked